lib/internal/util.js

```diff
@@ -186,4 +197,27 @@ exports.convertToValidSignal = function convertToValidSignal(signal) {
   }
 
   throw new Error('Unknown signal: ' + signal);
+}
+
+module.exports = {
```

Comment:
assigning to `exports = ` as well is more future proof, it allows util functions to refer to each other more easily.

Of the 3  common export styles, at-top, at-definition (what util used to do), and at-bottom,at-bottom remains my least favorite. It lacks the visibility and readability of export at top, and lacks the cohesion of export at time of definition. But if at-bottom is how node.js goes, I'll follow along.